command/node_status.go

```diff
@@ -149,9 +149,9 @@ func (c *NodeStatusCommand) Run(args []string) int {
 		// Format the nodes list
 		out := make([]string, len(nodes)+1)
 		if c.list_allocs {
-			out[0] = "ID|DC|Name|Class|Drain|Status|Running Allocs"
+			out[0] = "ID|DC|Name|Class|Build|Drain|Status|Running Allocs"
 		} else {
```

Comment:
Any thoughts on how to best accomplish this @dadgar? From my understanding verbose is currently only used to determine the length of the nodeID displayed. I am trying to avoid using numerous else ifs to reconcile the differences between the -allocs and -verbose flag usage:

`		if c.list_allocs && !c.verbose {
			out[0] = "ID|DC|Name|Class|Drain|Status|Running Allocs"
		} else if c.list_allocs && c.verbose {
			out[0] = "ID|DC|Name|Class|Build|Drain|Status|Running Allocs"
`